cryptosuite/jws2020/testutil_test.go

```diff
@@ -0,0 +1,32 @@
+package jws2020
+
+import (
+	"github.com/TBD54566975/ssi-sdk/crypto"
+)
+
+type TestCredential struct {
+	Context           any           `json:"@context" validate:"required"`
+	ID                string        `json:"id,omitempty"`
+	Identifier        string        `json:"identifier,omitempty"`
+	Type              any           `json:"type" validate:"required"`
+	Name              string        `json:"name,omitempty"`
+	Description       string        `json:"description,omitempty"`
+	Issuer            any           `json:"issuer" validate:"required"`
+	IssuanceDate      string        `json:"issuanceDate" validate:"required"`
+	ExpirationDate    string        `json:"expirationDate,omitempty"`
+	CredentialStatus  any           `json:"credentialStatus,omitempty" validate:"omitempty,dive"`
+	CredentialSubject any           `json:"credentialSubject" validate:"required"`
+	CredentialSchema  any           `json:"credentialSchema,omitempty" validate:"omitempty,dive"`
+	RefreshService    any           `json:"refreshService,omitempty" validate:"omitempty,dive"`
+	TermsOfUse        []any         `json:"termsOfUse,omitempty" validate:"omitempty,dive"`
+	Evidence          []any         `json:"evidence,omitempty" validate:"omitempty,dive"`
+	Proof             *crypto.Proof `json:"proof,omitempty"`
```

Comment:
I think this small change would fail tests. If so, then let's keep the issue open. 
```suggestion
	Proof             *crypto.Proof
```

Reply:
it would fail if there isn't a json property called `proof`

should I make an amending commit to rename the interface to `WithEmbeddedJSONProof`?